2025-07-11 05:43:03,704 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000224EFB4FFD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/32a6323f0f7600670c1c6d7d6d746866
2025-07-11 05:43:07,789 - urllib3.connectionpool - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000224E4758990>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/32a6323f0f7600670c1c6d7d6d746866
2025-07-11 05:43:11,875 - urllib3.connectionpool - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000224E4759350>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/32a6323f0f7600670c1c6d7d6d746866
2025-07-11 12:55:04,761 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None)': /session/9f7bdec338d7822359b778b82f196bfb
2025-07-11 12:55:08,821 - urllib3.connectionpool - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001D70D7E6D10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/9f7bdec338d7822359b778b82f196bfb
2025-07-11 12:55:12,874 - urllib3.connectionpool - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001D70D7E7210>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/9f7bdec338d7822359b778b82f196bfb
2025-07-11 14:11:40,427 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None)': /session/3887b19e4e89f00b7f6fcf0ab35070e3
2025-07-11 14:11:44,515 - urllib3.connectionpool - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002C75D506BD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/3887b19e4e89f00b7f6fcf0ab35070e3
2025-07-11 14:11:48,587 - urllib3.connectionpool - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002C75D545A50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/3887b19e4e89f00b7f6fcf0ab35070e3
